### Seminar 5 - Oppgave strategisk binding

En investorgruppe har kjøpt bryggen til Hjalmar Johansen på Sommarøy, og de planlegger å bygge hotell der. I forkant av dette gjør ledelsen på Sommarøy Hotell en vurdering om de skal utvide sin kapasitet for å avskrekke etablering av nytt hotell. Investorgruppen vil i forkant av en mulig etablering kunne observere denne kapasitetsinvesteringen. 
Etterspørselen etter hotellrom på Sommarøy er Q =5700 – P, hvor P er markedspris og Q er antall overnattinger. Ved etablering i markedet vil bedriftene ha en irreversibel investeringskostnad på F = 70 000. Kostnaden per enhet kapasitet (antall sengeplasser) som den etablerte bedriften kan investere i er k = 4 000, og marginalkostnaden opp til kapasitetsgrensen er c1 = 600. For kvantum utover kapasitetsgrensen vil marginalkostnaden være c1+k = 4 600. Den potensielle nykommeren vil måtte bygge opp sin kapasitet (antall sengeplasser) ved etablering og vil da ha marginalkostnad   c2 = 4 600.

Hva vil du anbefale ledelsen på Sommarøy Hotell å gjøre i forkant av en mulig nyetablering i hotellmarkedet? 




Vi bruker modell for strategiske investeringer i kapasitet av Spence (1977) & Dixit (1980) til å løse oppgaven

In [1]:
import sympy as sp
from sympy import *
import numpy as np
from matplotlib import pyplot as plt

Vi ser på en modell med to aktører; etablert bedrift (B1) og potensiell nykommer (B2), hvor vi har et dynamisk spill med Trinn 1: Etablert aktør velger å investere i kapasitet 𝐾. Trinn 2: Nykommer observere 𝐾, og velger etablering eller ikke. Bedriftene velger optimalt nivå på kvantum og kapasitet

In [2]:
def demand_1(q1):
           return (5700-q1-q2)

In [8]:
def demand_2(q2):
           return (5700-q1-q2)

In [9]:
def marginalrevenue_1(q1):
    return (5700-q2-2*q1)

In [10]:
def marginalrevenue_2(q2):
    return (5700-q1-2*q2)

## Strategi for tilpasning

Bedrift 1 vil tilpasse seg som Stackelberg leder på trinn 2, og velge kapasitet på trinn 1 som er lik det optimale kvantum i likevekt (𝑞1=𝑘)

In [11]:
# Otimal tilpasning for bedrift 2 der MR = MC (=60)
q2=sp.symbols('q2', real=True, positive=True)
q1=sp.symbols('q1', real=True, positive=True)
equ=sp.Eq(marginalrevenue_2(q2),4600)
equ

Eq(-q1 - 2*q2 + 5700, 4600)

In [12]:
#reaksjonsfunksjon til nykommer
q2_equ=sp.solve(equ,q2)[0]
q2_equ

550 - q1/2

Reaksjonsfunksjonen til nykommer settes inn i etterspørselen til den etablerte bedriften

In [13]:
d_demand_1=demand_1(q1).subs({q2:q2_equ})
d_demand_1

5150 - q1/2

In [15]:
def marginalrevenue1_RF2(q1):
    return (5150-q1)

In [16]:
# Optimal tilpasning for bedrift 1 der MR = MC (=30+30), 
q2=sp.symbols('q2', real=True, positive=True)
q1=sp.symbols('q1', real=True, positive=True)
equ=sp.Eq(marginalrevenue1_RF2(q1),4600)
equ

Eq(5150 - q1, 4600)

In [17]:
# optimalt kvantum for den etablerte bedriften, og da også optimalt nivå på investering av kapasitet (k)
q1RF2_equ=sp.solve(equ,q1)[0]
q1RF2_equ

550

Optimalt kvantum for nykommer finner vi ved å sette q1 = 550 inn i reaksjonsfunksjonen til nykommer

In [18]:
q2_equ2=q2_equ.subs({q1:q1RF2_equ})
q2_equ2

275

In [23]:
def profit_1(q1,q2):
    return ((5700-q1-q2)-4600)*q1-70000

In [24]:
# Profitt for etablert bedrift ved tilpasningsstrategi
profit_1(q1,q2).subs({q1:q1RF2_equ,q2:q2_equ2})

81250

In [25]:
def profit_2(q1,q2):
    return ((5700-q1-q2)-4600)*q2-70000

In [26]:
# Profitt for nykommer bedrift ved tilpasningsstrategi
profit_2(q1,q2).subs({q1:q1RF2_equ,q2:q2_equ2})

5625

## Strategi for avskrekking

For å avskrekke B2 fra å etablere seg må B1 på en troverdig måte binde seg til en kapasitet (på trinn1 ), slik at B2 ikke tjener noe ved å etablere seg i dette markedet. Altså, hvor stor må 𝒒𝟏 være for at 𝝅_𝟐 ≤ 0?

𝜋2=(5700−(𝑞1+𝑞2)-4600)𝑞2−70000 ≤ 0

In [27]:
# For å finne nivå på 𝑞1 som oppfyller likheten 𝜋2 = 0 settes R2 inn i uttrykket

profit_2(q1,q2).subs({q2:q2_equ})

(550 - q1/2)**2 - 70000

In [30]:
# Optimalt nivå på kapasitet for å avskrekke nykommer fra å etablere seg er lik:
q1_equ_k=sp.solve(profit_2(q1,q2).subs({q2:q2_equ}))[0]
q1_equ_k
round(q1_equ_k,2)

570.85

In [31]:
def profit_1(q1):
    return ((5700-q1)-4600)*q1-70000

In [37]:
# Profitt for etablert bedrift ved strategi for avskrekking:
round(profit_1(q1).subs({q1:q1_equ_k}))

232065

Det vil være optimalt å investere i kapasitet for å hindre nyetablering. Den etablerte bedriften vil da opptre som monopolist i markedet.